### ISSUE TITLE
Check-DB Initcontainer runs into endless timeout

### ISSUE THREAD

**Issue by yehlo** [airflow-helm/charts#181]
**What is the bug?**

The command to check the airflow db takes more than 16s sometimes. 
Funnily enough I have pods that just can't seem to boot up with 16s while others have no issue whatsoever. 
https://github.com/airflow-helm/charts/blob/6efcae8394055ec091980df7d1c1eec8e199a7e3/charts/airflow/templates/_helpers/pods.tpl#L29

I am using the k8s postgresql db. 
I checked our Grafana if there was a ressource bottleneck. It certainly doesn't seem like it. 

I then changed the value to 30s and the pods booted up without an issue. 
I think it would make sense to keep the default as is but add a configuration value to customize this according to the environment. 

**What are your Helm values?**

-

**What is your Kubernetes Version?**:

```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-03-25T14:58:59Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.2", GitCommit:"f5743093fd1c663cb0cbc89748f730662345d44d", GitTreeState:"clean", BuildDate:"2020-09-16T13:32:58Z", GoVersion:"go1.15", Compiler:"gc", Platform:"linux/amd64"}
```

**What is your Helm version?**:

```console
$ helm version
version.BuildInfo{Version:"v3.3.4", GitCommit:"a61ce5633af99708171414353ed49547cf05013d", GitTreeState:"clean", GoVersion:"go1.14.9"}
```

**Comment by thesuperzapper** [comment]
@yehlo this should be resolved after version `8.0.9` of the chart, as I have made the timeout 60s.

**Comment by anuragphadnis** [comment]
> @yehlo this should be resolved after version `8.0.9` of the chart, as I have made the timeout 60s.

Sorry for commenting on a closed issue but shouldn't the timeout be configurable, setting it to 60seconds will not solve this issue for everyone.
